client/src/main/java/org/apache/rocketmq/client/producer/DefaultMQProducer.java

```diff
@@ -197,20 +208,21 @@ public DefaultMQProducer(final String producerGroup, RPCHook rpcHook) {
     /**
      * Constructor specifying namespace, producer group and RPC hook.
      *
-     * @param namespace Namespace for this MQ Producer instance.
+     * @param namespace     Namespace for this MQ Producer instance.
      * @param producerGroup Producer group, see the name-sake field.
-     * @param rpcHook RPC hook to execute per each remoting command execution.
+     * @param rpcHook       RPC hook to execute per each remoting command execution.
      */
     public DefaultMQProducer(final String namespace, final String producerGroup, RPCHook rpcHook) {
         this.namespace = namespace;
         this.producerGroup = producerGroup;
         defaultMQProducerImpl = new DefaultMQProducerImpl(this, rpcHook);
+        produceAccumulator = MQClientManager.getInstance().getOrCreateProduceAccumulator(this);
```

Comment:
Should it be judged that autoBatch is true when creating produceAccumulator？

Reply:
```
producer.setAutoBatch(true);
producer.batchMaxDelayMs(xxx);
producer.batchMaxBytes(xxx);
producer.totalBatchMaxBytes(xxx);
```
The usage of the interface, after being exposed to the user, should support that he can call at any time